roofit/roofitcore/src/RooAddPdf.cxx

```diff
@@ -197,7 +177,7 @@ RooAddPdf::RooAddPdf(const char *name, const char *title, const RooArgList& inPd
   // Constructor with N PDFs and N or N-1 coefs
   RooArgList partinCoefList ;
 
-  Bool_t first(kTRUE) ;
+  Bool_t first(true) ;
```

Comment:
If you go towards standard c++ using `kTRUE --> true`, I would also use `Bool_t --> bool`.

Reply:
Okay!